README.md

```diff
@@ -120,7 +118,7 @@ The endpoint also accepts a batch of spans, which then need to be given as array
 
 ## Limitations
```

Comment:
We should point at https://www.instana.com/docs/tracing/custom-best-practices/ for guidance on which tags to use when

Reply:
Added that as point in the FAQ section.